packages/@o3r/logger/src/services/logger/logger.service.ts

```diff
@@ -16,6 +16,13 @@ export class LoggerService implements Logger {
   /** Loggers */
   private readonly clients: LoggerClient[];
 
+  /**
+   * Record the recording status to make sure that new clients recording status will be consistent with the service
+   *
+   * @private
```

Comment:
not needed

Reply:
auto-generated

Reply:
By your IDE? Maybe you should review a bit its configu